scripts/scrub.js

```diff
@@ -122,7 +122,11 @@ async function run() {
 
   // do these before deleting node_nodules
   console.log('\nClearing Jest cache...');
-  spawn(os.platform() === 'win32' ? 'npx.cmd' : 'npx', ['jest', '--clearCache']);
+  try {
```

Comment:
I have never seen this script being used, can we just remove it ?

Reply:
+1, this is a ticking bomb IMHO for anyone not familiar with this monorepo setup - removing untracked files etc should be done manually by git by current user.

Reply:
lets discuss this during @microsoft/fluentui-react-build  sync

Reply:
This is never run automatically. It's intended to help recover the repo from a bad state, in a way compatible with Windows (IIRC `git clean -fdx` sometimes incorrectly handles symlinks on Windows).

Reply:
How is this a ticking time bomb when it is not run automatically *and* explicitly warns that it's going to delete untracked files and requires confirmation when you start the script??